client/src/components/ProfileDetail/ProfileDetail.tsx

```diff
@@ -0,0 +1,58 @@
+import { Avatar, Card, CircularProgress, Grid, Paper, Typography } from '@mui/material';
+import { useAuth } from '../../context/useAuthContext';
+import React from 'react';
+import { useStyles } from './useStyles';
+import { LocationOn } from '@mui/icons-material';
+
+const ProfileDetail = (): JSX.Element => {
+  const classes = useStyles();
+  const { profile } = useAuth();
+
+  const title = () => {
+    return profile.accountType === 'pet_owner' ? 'Kind pet owner' : 'Loving pet sitter';
+  };
+
+  console.log(profile);
```

Comment:
Please watch for unnecessary logs to the console. Going forward, we should be cleaning up our logs done during development. Thanks!